BlazarUI/app/scripts/components/branch/BranchContainer.jsx

```diff
@@ -105,10 +122,16 @@ class BranchContainer extends Component {
             </UIGridItem>
             <UIGridItem size={2} align='RIGHT'>
               <BuildButton 
-                triggerBuild={this.triggerBuild} 
+                triggerBuild={this.openModuleModal.bind(this)} 
```

Comment:
you can simplify/keep all these bindings clean and packaged up by using `_.bindAll` in the constructor - you can see examples of this across the app.